src/main/java/ch/njol/skript/expressions/ExprLastSpawnedEntity.java

```diff
@@ -43,44 +45,48 @@
 import ch.njol.skript.lang.util.SimpleExpression;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Last Spawned Entity")
 @Description("Holds the entity that was spawned most recently with the spawn effect (section), dropped with the <a href='../effects/#EffDrop'>drop effect</a>, shot with the <a href='../effects/#EffShoot'>shoot effect</a> or created with the <a href='../effects/#EffLightning'>lightning effect</a>. " +
 		"Please note that even though you can spawn multiple mobs simultaneously (e.g. with 'spawn 5 creepers'), only the last spawned mob is saved and can be used. " +
 		"If you spawn an entity, shoot a projectile and drop an item you can however access all them together.")
-@Examples({"spawn a priest",
-		"set {healer::%spawned priest%} to true",
-		"shoot an arrow from the last spawned entity",
-		"ignite the shot projectile",
-		"drop a diamond sword",
-		"push last dropped item upwards",
-		"teleport player to last struck lightning"})
-@Since("1.3 (spawned entity), 2.0 (shot entity), 2.2-dev26 (dropped item), SINCE VERSION (struck lightning)")
+@Examples({
+	"spawn a priest",
+	"set {healer::%spawned priest%} to true",
+	"shoot an arrow from the last spawned entity",
+	"ignite the shot projectile",
+	"drop a diamond sword",
+	"push last dropped item upwards",
+	"teleport player to last struck lightning",
+	"delete last launched firework"
+})
+@Since("1.3 (spawned entity), 2.0 (shot entity), 2.2-dev26 (dropped item), INSERT VERSION (struck lightning, firework)")
 public class ExprLastSpawnedEntity extends SimpleExpression<Entity> {
 	
 	static {
 		Skript.registerExpression(ExprLastSpawnedEntity.class, Entity.class, ExpressionType.SIMPLE,
 			"[the] [last[ly]] (0:spawned|1:shot) %*entitydata%",
 			"[the] [last[ly]] dropped (2:item)",
-			"[the] [last[ly]] (created|struck) (3:lightning)");
+			"[the] [last[ly]] (created|struck) (3:lightning)",
+			"[the] [last[ly]] launched (4:firework)");
```

Comment:
```suggestion
			"[the] [last[ly]] (launched|deployed) (4:firework)");
```
to match with the syntax of EffFireworkLaunch